warehouse/templates/pages/help.html

```diff
@@ -123,6 +124,7 @@ <h2><a href="#my-account">{% trans %}My Account{% endtrans %}</a></h2>
           <li><a href="#verified-email">{{ verified_email() }}</a></li>
           <li><a href="#compromised-password">{{ compromised_password() }}</a></li>
           <li><a href="#suspicious-activity">{{ suspicious_activity() }}</a></li>
+          <li><a href="#compromised-token">{{ compromised_token() }}</a></li>
```

Comment:
It looks like this help section never got written. A shorter version of the notification email should suffice.